Src/Support/Google.Apis.Tests/Apis/Requests/ClientServiceRequestTest.cs

```diff
@@ -654,9 +651,7 @@ public void Execute_Error()
 
                 Assert.Equal(1, handler.Calls);
                 Assert.Equal(ErrorMessageHanlder.ExpectedExceptionMessage, exception.Message);
-                string exToString = exception.ToString();
-                Assert.Contains(ErrorMessageHanlder.ExpectedError, exToString);
-                Assert.Contains(ErrorMessageHanlder.ErrorContent, exToString);
+                Assert.Contains(ErrorMessageHanlder.ExpectedError, exception.ToString());
```

Comment:
Nit: at some point, maybe not in this PR, let's fix `ErrorMessageHanlder` to `ErrorMessageHandler`.